typescript-getting-started/firebase.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file here so one can use inline commands? e.g. `firebase deploy --project foo`?

Reply:
`firebase init` creates this file

Reply:
Sure; but if `firebase init` is part of the setup instructions then we don't need this. Not a huge deal either way.